source/getting-started/customizing-devices.markdown

```diff
@@ -11,6 +11,8 @@ footer: true
 
 By default, all of your devices will be visible and have a default icon determined by their domain. You can customize the look and feel of your front page by altering some of these parameters. This can be done by overriding attributes of specific entities.
 
+`customize` consists of a list of attribute customization blocks
```

Comment:
I wonder if it makes sense to keep the old mode around? Maybe something to ask in the chat channel?